### DISCUSSION TITLE
Force a larger page width on print

### DISCUSSION THREAD

**Discussion by volodymyrprokopyuk** [tailwindlabs/tailwindcss#13944] [Help]
Hi,
I've designed a CV with a mobile (up until `lg` breakpoint) and a desktop (from `lg` onwards) layouts at https://volodymyrprokopyuk.github.io/cv/. The automatic layout selection works correctly. When I try to print a CV to PDF from a desktop layout on my laptop I unexpectedly get the mobile layout for the A4 paper. When I select the A3 (larger) paper I get the expected desktop layout of a CV. How can I force the desktop layout for the A4 paper?

I've tried `<body class="print:w-[1280px]">` without success: the layout is still mobile, only the text got smaller.

Thank you!

**Comment by Pablushka** [comment]
Hi Volodymyr. I've the same problem an tried almost everything... No success.

**Reply by volodymyrprokopyuk** [reply]
Manually changing the format from A4 to A3 before printing to PDF worked for me.
